### PR TITLE
grpc argument

### PR DESCRIPTION
**Pull request by juniuszhou**
We need add the sifnode grpc entry point as an argument for relayer.

### CODE REVIEW

File: cmd/ebrelayer/main.go [hunk 1]
```diff
@@ -110,7 +112,7 @@ func buildRootCmd() *cobra.Command {
 func initRelayerCmd() *cobra.Command {
 	//nolint:lll
 	initRelayerCmd := &cobra.Command{
-		Use:     "init-relayer --network-descriptor 1 --tendermint-node tcp://localhost:26657 --web3-provider ws://localhost:7545/ --bridge-registry-contract-address 0x --validator-mnemonic mnemonic --chain-id=peggy",
+		Use:     "init-relayer --network-descriptor 1 --tendermint-node tcp://localhost:26657 --web3-provider ws://localhost:7545/ --bridge-registry-contract-address 0x --validator-mnemonic mnemonic --chain-id=peggy --sifnode-grpc 0.0.0.0:9090",
```
Comment (jprupp) [inline cmd/ebrelayer/main.go:115]: You made me maximise by browser window.

File: cmd/ebrelayer/main.go [hunk 2]
```diff
@@ -287,6 +299,7 @@ func RunInitRelayerCmd(cmd *cobra.Command, args []string) error {
 		web3Provider,
 		contractAddress,
 		sugaredLogger,
+		sifnodeGrpc,
```
Comment (jprupp) [inline cmd/ebrelayer/main.go:302]: It's so good when you can put a comma at the end of a list before before the closing bracket. I'm looking at you JSON.

**Review by jprupp — APPROVED** [review]
Looks good.

File: cmd/ebrelayer/main.go [hunk 3]
```diff
@@ -401,6 +415,16 @@ func RunInitWitnessCmd(cmd *cobra.Command, args []string) error {
 		return errors.New("validator moniker is empty")
 	}
 
+	sifnodeGrpc, err := cmd.Flags().GetString(sifnodeGrpcEntryPointFlag)
+	if err != nil {
+		// set default value
+		sifnodeGrpc = defaultSifnodeGrpc
+	}
+
+	if len(sifnodeGrpc) == 0 {
```
Comment (smartyalgo) [inline cmd/ebrelayer/main.go:424]: When will sifnodeGrpc be 0 length?

**Review by smartyalgo — APPROVED** [review]

File: cmd/ebrelayer/main.go (hunk 3 above)
Comment (juniuszhou) [reply cmd/ebrelayer/main.go:424]: input is a empty string like --sifnode-grpc ""

**Pull request merged**